setup.md

```diff
@@ -7,33 +7,51 @@ Please seek help at the start of the lesson if you have not been able to establi
 
 ### Files to download
 
-Download the `docker-intro.zip` file from the main GitHub repository page: 
-[Docker Introduction Github](https://github.com/carpentries-incubator/docker-introduction/). 
+Download the [`docker-intro.zip`]({{ page.root }}/files/docker-intro.zip) file. _This file can alternatively be downloaded from the `files` directory in the [docker-introduction GitHub repository](https://github.com/carpentries-incubator/docker-introduction/)_.
 
 Move the downloaded file to your Desktop and unzip it. It should unzip to a folder called `docker-intro`. 
 
-### Software to install
-Unfortunately, in many common situations installing Docker on your laptop will not straightforward if you do not have a large amount of technical experience. We have helpers on hand that have worked their way through the install process but be prepared for some troubleshooting.
+### Software to install: Docker
+
+Docker can be installed on Windows 10, macOS and Linux. However, in some cases, installing Docker on your laptop may, unfortunately, not be completely straightforward if you do not have significant technical experience. We have helpers on hand that have worked their way through the install process but be prepared for some troubleshooting.
 
 Please try to install the appropriate software from the list below depending on the operating system that your laptop is running:
-#### Microsoft Windows
-**You must have admin rights to run docker!** Some parts of the lesson will work without running as admin but if you are unable to `Run as admin` on your machine some of this workshop might not work easily.
 
-If you have Windows 10 Pro Edition:
- - First try to install the [Docker Desktop (Windows)](https://hub.docker.com/editions/community/docker-ce-desktop-windows), or **failing that**;
- - Install the [Docker Toolbox (Windows)](https://docs.docker.com/toolbox/toolbox_install_windows/).
+#### **Microsoft Windows**
+**You must have admin rights to run docker!** Some parts of the lesson will work without running as admin but if you are unable to `Run as administrator` on your machine some elements of this workshop might not work as described.
+
+On Windows 10, you will need to install [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows). 
+
+_Previously Windows 10 Home users needed to install an alternative version of Docker - [Docker Toolbox](https://docs.docker.com/docker-for-windows/docker-toolbox/) - however, this is now deprecated. Docker Desktop for Windows can now also be installed on Windows 10 Home Edition_
```

Comment:
I suggest we remove this line.  I don't think people installing for the first time will need to know this.

Reply:
I think the section below on using toolbox as backup if you can't get docker desktop covers it well.

Reply:
Yes, this seems reasonable, I've removed the sentence starting "Previously Windows 10 Home users..."